test/iron-location.html

```diff
@@ -201,18 +201,25 @@
           'foo#bar': '?foo%23bar',
           'foo?bar': '?foo?bar',
           '/foo\'bar\'baz': ['?/foo%27bar%27baz', '?/foo\'bar\'baz'],
+          'foo/bar/baz': '?foo/bar/baz'
         };
         for (var plainTextQuery in queryEncodingExamples) {
           var encodedQueries = queryEncodingExamples[plainTextQuery];
+          var ironLocationQuery = encodedQueries;
           if (typeof encodedQueries === 'string') {
             encodedQueries = [encodedQueries];
+            ironLocationQuery = [ironLocationQuery.substring(1)];
+          } else {
+            ironLocationQuery = ironLocationQuery.map(function(value) {
+              return value.substring(1);
+            });
           }
 
           expect(urlElem._initialized).to.be.eq(true);
           urlElem.query = plainTextQuery;
           expect(encodedQueries).to.contain(window.location.search);
-          expect(urlElem.query).to.be.equal(plainTextQuery);
-          expect(makeTemporaryIronLocation().query).to.be.equal(plainTextQuery);
+          expect(ironLocationQuery).to.contain(urlElem.query);
+          expect(ironLocationQuery).to.contain(makeTemporaryIronLocation().query);
```

Comment:
Another thing to assert here is to set `urlElem.query` to a dummy value, then set it to `ironLocationQuery[0]` and then do these same three assertions again.